Source/WebCore/DerivedSources.make

```diff
@@ -2088,6 +2090,20 @@ $(MATH_ML_GENERATED_PATTERNS) : $(WebCore)/dom/make_names.pl $(WebCore)/bindings
 
 # --------
 
+# All known DOM names
+
+DOM_NAMES_GENERATED_FILES = \
+    DOMNames.cpp \
+    DOMNames.h \
```

Comment:
Do we really need a single file that contains all attribute names & element names?

Reply:
As discussed, I'll split the generated files into separate TagName.h/cpp, ElementName.h/cpp, and Namespace.h/cpp files.